v2/Dockerfile

```diff
@@ -1,4 +1,43 @@
-FROM quay.io/cybozu/ubuntu:20.04
+FROM --platform=$TARGETPLATFORM golang:1.18-buster as build-env
```

Comment:
Tip: you might want to use `$BUILDPLATFORM`
Go builds pretty fast so in this case it may not speed things up that much, but since go supports cross-compilation you're better off using the same platform as your build environment (GHA) rather than an emulated (qemu) platform.

Reply:
Thank you. I fixed it.